translations/en/LC_MESSAGES/messages.po

```diff
@@ -349,7 +349,7 @@ msgid "classes"
 msgstr "Classes"
 
 msgid "classes_info"
-msgstr "Create a class to follow the progress of each student in dashboard, and to customize the adventures your students see, and even adding your own! You can create as many classes as you like, and each class can have multiple teachers each one with different roles. You can also add as many students as you want, but mind that each student can only be in one class at a time. You can find more information about classes in the <a href=\"https://hedy.org/for-teachers/manual/preparations#for-teachers\">teacher manual</a>."
+msgstr "Create a class to follow the progress of each student in dashboard, and to customize the adventures your students see, and even adding your own! You can create as many classes as you like, and each class can have multiple teachers each one with different roles. You can also add as many students as you want, but mind that each student can only be in one class at a time. You can find more information about classes in the <a #HREF_TEACHERS_MANUAL_FOR_TEACHERS#>teacher manual</a>."
```

Comment:
I think we should surround these in `{}`, because if they change this it'd mess up their translation. You're not supposed to translate things that are surrounded with those!

Reply:
Excellent idea, thanks!